homeassistant/components/tile/coordinator.py

```diff
@@ -26,6 +32,7 @@ def __init__(self, hass: HomeAssistant, client: API, tile: Tile) -> None:
         )
         self.tile = tile
         self.client = client
+        self.username = self.config_entry.data[CONF_USERNAME]
```

Comment:
Why is this necessary?

Reply:
Because I don't really feel like doing `coordinator.config_entry.data[CONF_USERNAME]` in the unique id attribute and this makes it a bit shorter